### PR TITLE
Feat: manually inject content script

### PR DESCRIPTION
**Pull request by karliatto**
## Description

* 5818e22fb151af50be00f56234f3917cff3fa3ea - include content-script.js into build to be publish in the package so 3rd party integration could inject it manually in order not to need `scripting` permissions in the webextension.
* b92f81bea84db43bec157207eec8eb8005cd276b - checks permissions for `scripting` before trying to do it.

## Related Issue

Resolve

### CODE REVIEW

File: packages/connect-webextension/package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
         "@trezor/connect": "workspace:*",
         "@trezor/connect-common": "workspace:*",
         "@trezor/utils": "workspace:*",
+        "copy-webpack-plugin": "^12.0.1",
```
Comment (mroz22) [inline packages/connect-webextension/package.json]: shouldn't this be dev dependecy?

File: packages/connect-webextension/src/popup.ts [hunk 2]
```diff
@@ -114,16 +114,22 @@ export class PopupManager extends EventEmitter {
...
+                        target: { tabId },
+                        // content script is injected into body of func in build time.
+                        func: () => {
+                            // <!--content-script-->
+                        },
+                    })
+                    .then(() => this.logger.debug('content script injected'))
+                    .catch(error => this.logger.error('content script injection error', error));
+            } else {
+                // When permissions for `scripting` are not provided 3rd party integrations have include content-script.js manually.
```
Comment (mroz22) [inline packages/connect-webextension/src/popup.ts:130]: nice solution with permissions check!

**Review by mroz22 — COMMENTED** [review]
please try to expand documentation on how it works, what kind of options 3rd party devs have. 🙏

File: packages/connect-webextension/package.json (hunk 1 above)
Comment (karliatto) [reply packages/connect-webextension/package.json]: Yes, done.

**Comment by karliatto** [comment]
> please try to expand documentation on how it works, what kind of options 3rd party devs have. 🙏

@mroz22 Included some documentation about the current 2 options we offer to use this package [29f0c64](https://github.com/trezor/trezor-suite/pull/10666/commits/29f0c64e9fd57087610d8cb1bfd6750bccb691d2)

**Review by mroz22 — APPROVED** [review]
looks good, lets try!

**Pull request merged**